config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,55 +3,35 @@
         "5.1":{
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "",
+                "cves": "CVE-2022-48560, CVE-2022-48564, CVE-2023-39615, CVE-2023-43804, CVE-2023-45803, CVE-2023-5981",
+                "golang": "1.20.11",
+                "ubi": "8.9-1108",
+                "new-features": [
+                    "Updated Kubernetes dependency to 1.28",
+                    "Updated csi node driver registrar image to v2.9.3",
+                    "Updated livenessprobe image to v2.11.0",
+                    "Updated csi-snapshotter image to v6.3.3",
+                    "Updated csi-provisioner image to v3.6.3",
+                    "Updated csi-attacher image to v4.4.3",
+                    "Updated csi-resizer image to v1.9.3"
```

Comment:
should these come under `fixes` or `new features`?

Reply:
Also even if `fixes` field is empty, dont remove it. Doc team asked to retain fields with even empty values.